<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>520</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Fixedsys</family>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Dungeon Slayer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>400</width>
      <height>500</height>
     </rect>
    </property>
    <widget class="QWidget" name="main_menu">
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>160</y>
        <width>130</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="StartButton">
         <property name="font">
          <font>
           <family>Fixedsys</family>
          </font>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ExitButton">
         <property name="font">
          <font>
           <family>Fixedsys</family>
          </font>
         </property>
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="GameName">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>150</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Fixedsys</family>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Dungeon Slayer</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="character_edit">
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>300</width>
        <height>431</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QLabel" name="AllocateStats">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Fixedsys</family>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Allocate Stats</string>
       </property>
      </widget>
      <widget class="QPushButton" name="BackButton">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>5</y>
         <width>40</width>
         <height>26</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Fixedsys</family>
        </font>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
      <widget class="QLabel" name="Class">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>44</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Class</string>
       </property>
      </widget>
      <widget class="QComboBox" name="Classes">
       <property name="geometry">
        <rect>
         <x>58</x>
         <y>40</y>
         <width>241</width>
         <height>26</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string notr="true">Barbarian</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Beastmaster</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Knight</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Paladin</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Rogue</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="ClassSkill">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>281</width>
         <height>51</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Fixedsys</family>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LayoutDirection::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Test text. It should be in separeted lines i guess... Right? Right now.</string>
       </property>
       <property name="textFormat">
        <enum>Qt::TextFormat::PlainText</enum>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignJustify|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="HealthIcon">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>assets/status/health_icon.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>130</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>HP: 100</string>
       </property>
      </widget>
      <widget class="QLabel" name="HealthIcon_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>assets/status/damage_icon2.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>150</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ATK: 100</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>190</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>APS: 0.5</string>
       </property>
      </widget>
      <widget class="QLabel" name="HealthIcon_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>assets/status/defense_icon2.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>170</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>DEF: 100</string>
       </property>
      </widget>
      <widget class="QLabel" name="HealthIcon_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>190</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>assets/status/aps_icon.png</pixmap>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>